### PR TITLE
Add PriorBoxClustered and tests.

### PR DESCRIPTION
**Pull request by XinWangIntel**
Signed-off-by: xin1.wang <xin1.wang@intel.com>

### CODE REVIEW

**Comment by XinWangIntel** [comment]
Depend on https://github.com/Flex-plaidml-team/openvino/pull/38

File: plaidml/bridge/openvino/ops/prior_box_clustered.cpp [hunk 1]
```diff
@@ -0,0 +1,120 @@
+// Copyright (C) 2020 Intel Corporation
```
Comment (YangleiZouIntel) [inline plaidml/bridge/openvino/ops/prior_box_clustered.cpp]: Please update to 2021.

File: plaidml/bridge/openvino/ops/prior_box_clustered.cpp [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    IE_ASSERT(attrs.variances.size() == 1 || attrs.variances.size() == 4 || attrs.variances.empty());
+
+    auto precision = to_plaidml(ctx.layer->get_output_element_type(0));
+    int64_t num_priors = static_cast<int64_t>(attrs.widths.size());
+    auto variances = attrs.variances;
+    if (variances.empty()) variances.push_back(0.1f);
+
+    auto input0_shape = cast_constant_operand<int64_t>(0, layer);
+    auto input1_shape = cast_constant_operand<int64_t>(1, layer);
+    int64_t layer_height = static_cast<int64_t>(input0_shape[0]);
```
Comment (YangleiZouIntel) [inline plaidml/bridge/openvino/ops/prior_box_clustered.cpp]: Are these static casts to int64_t necessary?

**Comment by YangleiZouIntel** [comment]
LGTM. How long does the test take?

File: plaidml/bridge/openvino/ops/prior_box_clustered.cpp (hunk 1 above)
Comment (XinWangIntel) [reply plaidml/bridge/openvino/ops/prior_box_clustered.cpp]: Done

File: plaidml/bridge/openvino/ops/prior_box_clustered.cpp (hunk 2 above)
Comment (XinWangIntel) [reply plaidml/bridge/openvino/ops/prior_box_clustered.cpp]: only inputx_shape need to cast, remove others

**Comment by XinWangIntel** [comment]
> LGTM. How long does the test take?

Each test takes 0.2-0.4s now

**Comment by XinWangIntel** [comment]
Close, track with https://github.com/plaidml/plaidml/pull/1689 now

**Pull request closed without merging**